src/util/options.schema.json

```diff
@@ -2102,6 +2102,12 @@
                 "Whether to dump assertions about all local variables or limitting it to modified ones where possible.",
               "type": "boolean",
               "default": true
+            },
+            "blobs": {
+              "title": "witness.invariant.blobs",
```

Comment:
I think it would be more appropriate to call this `ana.base.invariant.blob` because blobs are a base analysis specific thing.